Source/WebCore/inspector/FrameInspectorController.cpp

```diff
@@ -110,7 +110,7 @@ void FrameInspectorController::createLazyAgents()
 
     m_injectedScriptManager->connect();
     if (auto& commandLineAPIHost = m_injectedScriptManager->commandLineAPIHost())
-        commandLineAPIHost->init(m_instrumentingAgents.copyRef());
+        commandLineAPIHost->init(m_instrumentingAgents.ptr());
```

Comment:
If m_instrumentingAgents is gone, then `.ptr()` will `RELEASE_ASSERT` here.

I am unsure why this became pointer-based. We don't initialize it to nullptr anywhere I see. Can we pass by reference `InstrumentingAgents&` and bail out if the WeakPtr is somehow gone at this point?

Reply:
yeah we should definitely use `InstrumentingAgents&` as nobody should ever pass `nullptr`